sample.js

```diff
@@ -0,0 +1,4 @@
+
+function sampleFunction() {
```

Comment:
This is a PR commetn:

![](https://github.com/mlewand/foobar/assets/5353898/9c6e0347-4015-4836-bf5e-4b98d0a38a7a)

aa